vite.config.ts

```diff
@@ -12,12 +18,29 @@ export default defineConfig({
     },
   },
   plugins: [
-    react(), 
-    svgr(),
+    react(),
+    svgr({
+      include: '**/*.svg',
+      svgrOptions: {
+        exportType: 'default',
+      },
+    }),
     dts(),
+    visualizer({
+      filename: './dist/report.html',
+      brotliSize: true,
+    }),
+    // pluginPurgeCss({
+    //   variables: true,
+    // }),
   ],
-  define: {
-    APP_VERSION: JSON.stringify(process.env.npm_package_version),
+  resolve: {
+    alias: [
+      {
+        find: '@uikit',
+        replacement: path.resolve(__dirname, 'packages/uikit/src'),
+      },
+    ],
```

Comment:
what is resolve for?

Reply:
Also related to https://github.com/sendbird/chat-ai-widget/pull/180#discussion_r1584204401